website/client/components/tasks/tagsPopup.vue

```diff
@@ -0,0 +1,85 @@
+<template lang="pug">
+.tags-popup
+  .tags-category.d-flex
+    .tags-header
+      strong(v-once) {{ $t('tags') }}
+    .tags-list.container
+      .row
+        template
```

Comment:
what's `template` here for?

Reply:
I've mostly copied this structure from the tags list in the filter popup in the User component, so some things might not be necessary.

I'm going to have a full run-through of the code at some point and clean up anything that isn't needed.